addOns/pscanrulesAlpha/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
   - Site Isolation
   - Sub Resource Integrity
 - Maintenance changes.
+- Renamed Feature Policy to Permissions Policy following Mozilla's change
```

Comment:
This should be tweaked to be generic, it's not just Mozilla's (webappsec-permissions-policy#359).

Reply:
Same for PR title.